src/main/java/frc/robot/subsystems/TiltedElevatorSubsystem.java

```diff
@@ -142,10 +173,31 @@ public TiltedElevatorSubsystem() {
         extensionFollow.setIdleMode(IdleMode.kBrake);
     }
 
+    
     @Override
     public void periodic() {
-        // if (zeroLimitSwitch.get()) extensionEncoder.setPosition(0); 
+        // Update elevator encoder using absolute magnet position
+        if (!hallEffectSensor.get()) {
+            // Use direction of elevator motion to set state of hall effect sensor
+            if (extensionEncoder.getPosition() > targetExtension) { // elevator moving down
```

Comment:
this would check against the target extension of the previous cycle, which might not be problematic but doesn't seem quite right. I don't think targetExtension should be a field at all, I think it should be scoped to this method

Reply:
also, i think this would trigger every periodic cycle that the hall effect sensor is on

Reply:
i think this is outdated now